external/storm-hdfs/README.md

```diff
@@ -159,6 +159,8 @@ For example:
 By default, prefix is empty and extenstion is ".txt".
 
 
+The provided `org.apache.storm.hdfs.format.SimpleFileNameFormat` is flexible, supports parameters such as `$TIME` `$NUM` `$HOST` etc.
```

Comment:
It may be better to elaborate how to use so that users don't need to look at javadoc.